worker-api/src/main/java/com/hpe/caf/api/worker/WorkerConfiguration.java

```diff
@@ -43,4 +45,12 @@ public void setWorkerVersion(String workerVersion)
     {
         this.workerVersion = workerVersion;
     }
+
+    public boolean isTrackProgressMessages() {
+        return trackProgressMessages;
+    }
+
+    public void setTrackProgressMessages(boolean trackProgressMessages) {
+        this.trackProgressMessages = trackProgressMessages;
+    }
```

Comment:
I know you talked to me about this but on reflection I don't think we should change this class.  Changing it means that we will have to update all the WorkerConfiguration.js files in each container.

Let's just update the code to use `System.getenv("CAF_WORKER_DISABLE_ZERO_PROGRESS_REPORTING")` from the code instead.  Overall it will reduce the number of changes that we need to make.